great_expectations/experimental/datasources/pandas_filesystem_datasource.py

```diff
@@ -140,12 +84,24 @@ def add_csv_asset(
         asset = CSVAsset(
             name=name,
             batching_regex=batching_regex_pattern,
-            glob_directive=glob_directive,
             order_by=order_by_sorters,
             **kwargs,
         )
 
-        return self.add_asset(asset)
+        data_connector: DataConnector = FilesystemDataConnector(
+            datasource_name=self.name,
+            data_asset_name=name,
+            batching_regex=batching_regex_pattern,
+            base_directory=self.base_directory,
+            glob_directive=glob_directive,
+            data_context_root_directory=self.data_context_root_directory,
+        )
+        test_connection_error_message: str = f"""No file at base_directory path "{self.base_directory.resolve()}" matched regular expressions pattern "{batching_regex_pattern.pattern}" and/or glob_directive "{glob_directive}" for DataAsset "{name}"."""
```

Comment:
`non-blocking`
This seems like the kind of thing that could be defined as an attribute on the `FilesystemDataConnector` class itself.
Then we wouldn't need to pass it.

Maybe not though 🤔

Reply:
@Kilo59 Interesting idea, and technically -- obviously -- easy to do.  However, the `DataConnector` only does the low-level access work; it does not have the responsibility to manage helpful messages to users.  In addition, turning this message, albeit tiny in terms of computation work, is borderline not a good fit for a property (properties must not build or compute anything; they should just retrieve a result, with only minor work, best avoided entirely).  Thanks.